packages/callout-block/src/CalloutBlock.tsx

```diff
@@ -12,16 +12,40 @@ import {
     radiusStyleMap,
     setAlpha,
 } from '@frontify/guideline-blocks-settings';
-import { CSSProperties, ReactElement } from 'react';
+import { CSSProperties, ReactElement, useEffect, useState } from 'react';
 import 'tailwindcss/tailwind.css';
 import '@frontify/guideline-blocks-settings/styles';
 import { CalloutIcon } from './components/CalloutIcon';
 import { getTextColor } from './helpers/getTextColor';
 import { ICON_ASSET_ID } from './settings';
 import { Appearance, BlockSettings, Icon, Type, Width, alignmentMap, outerWidthMap, paddingMap } from './types';
 
+const getAccentColor = (type: Type): string => {
+    const style = getComputedStyle(document.body);
+    switch (type) {
+        case Type.Info:
+            return style.getPropertyValue(`${THEME_PREFIX}accent-color-info-color`);
+        case Type.Note:
+            return style.getPropertyValue(`${THEME_PREFIX}accent-color-note-color`);
+        case Type.Tip:
+            return style.getPropertyValue(`${THEME_PREFIX}accent-color-tip-color`);
+        case Type.Warning:
+            return style.getPropertyValue(`${THEME_PREFIX}accent-color-warning-color`);
+    }
+};
+
+const computeStyles = (type: Type, appearance: Appearance) => {
```

Comment:
Can functions outside of the component move to helpers folder, similar to getTextColor? Maybe create a color.ts file that holds all the color related helper functions.